taskscheduler/doc.go

```diff
@@ -2,6 +2,12 @@
 Package taskscheduler provides a concurrent task execution framework for Go applications.
 
 This package enables you to schedule and execute tasks concurrently.
+
+Guidelines of this package are that you can call the Run Function only once and
```

Comment:
```suggestion
The guidelines of this package include:
- Each scheduler can be run only once.
- Tasks must be registered before their scheduler starts running.
- Stop() can be called only after Run() was called.
```

The better practice when writing doc of packages is not avoid using bullet points but in this case it improves the readability. 
For example the sim management doc [file](https://github.com/OneLayerHQ/atmosphere/blob/b82cc1b3d72bbe8344ca870e3f54bef4b2297ae5/simmgmt/doc.go).

Reply:
Fixed.